### PR TITLE
Add another possible combination of arguments for jQuery's .get() and…

### PR DESCRIPTION
**Pull request by ChrisRaven**
As jQuery [documentation](https://api.jquery.com/jquery.post/) says, only the url argument is obligatory, the rest are optional. The current definitions of the .get() and the post() types don't contain a combination, where there's a url, data and dataType, so I'm proposing to add it with this PR.

I had such combination of arguments in one of my working projects and it was underlined as an incorrect type. After adding the new definitions, everything was ok.

### CODE REVIEW

**Comment by ChrisRaven** [comment]
According to the checks, my addition should be merged with an already existing type definition. Technically, it's true. However, the parameters in both definitions have totally different meanings (callback vs data to send to the server). Not sure, what to do about it.

**Comment by martin-badin** [comment]
please add tests for all variants of your updated methods

File: types/jquery/JQueryStatic.d.ts [hunk 1]
```diff
@@ -1611,6 +1611,21 @@ $.get( "test.php" );
...
+     * @param data A plain object or string that is sent to the server with the request.
+     * @param dataType The type of data expected from the server. Default: Intelligent Guess (xml, json, script, text, html).
+     * @see \`{@link https://api.jquery.com/jQuery.get/ }\`
+     * @since 1.0
+     * @example ​ ````Request the test.php page, send an object with data, receive a json response.
+```javascript
+$.get( "test.php", { name: 'John' }, 'json' );
+```
+     */
+    get(url: string,
```
Comment (martin-badin) [inline types/jquery/JQueryStatic.d.ts:1626]: remove your definition and update this [get](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cd0e972f17bf5c892799692760828abe615430ee/types/jquery/JQueryStatic.d.ts#L1559-L1561) to 
```
get(url: string,
        data_success: JQuery.PlainObject | string | JQuery.jqXHR.DoneCallback | null,
        dataType: string): JQuery.jqXHR;
```

File: types/jquery/JQueryStatic.d.ts [hunk 2]
```diff
@@ -2761,6 +2776,21 @@ $.post( "test.php" );
...
+     * @param data A plain object or string that is sent to the server with the request.
+     * @param dataType The type of data expected from the server. Default: Intelligent Guess (xml, json, script, text, html).
+     * @see \`{@link https://api.jquery.com/jQuery.post/ }\`
+     * @since 1.0
+     * @example ​ ````Request the test.php page, send an object with data, receive a json response.
+```javascript
+$.post( "test.php", { name: 'John' }, 'json' );
+```
+     */
+    post(url: string,
```
Comment (martin-badin) [inline types/jquery/JQueryStatic.d.ts:2791]: remove your definition and update this [post](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cd0e972f17bf5c892799692760828abe615430ee/types/jquery/JQueryStatic.d.ts#L2676-L2678) to 
```
post(url: string,
         data_success: JQuery.PlainObject | string | JQuery.jqXHR.DoneCallback | null,
         dataType: string): JQuery.jqXHR;
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Pull request merged**